Traceback (most recent call last):
  File "C:\Python27\Scripts\env\TAS_Ksiegarnia\manage.py", line 8, in <module>
    from django.core.management import execute_from_command_line
ImportError: No module named django.core.management
